test/manual-test-examples/webgl/customGeometry/sketch.js

```diff
@@ -63,6 +66,6 @@ function draw() {
   pointLight(150, 150, 150, moonPos);
 
   geometry.computeFaces().computeNormals();
-  renderer.createBuffers("!", geometry);
-  renderer.drawBuffersScaled("!", 1000, 1000, 1);
+  renderer.createBuffers('!', geometry);
+  renderer.drawBuffersScaled('!', 1000, 1000, 1);
```

Comment:
oops, forgot to comment on this initially: this is fine, but for future PRs please only update the test examples if they're relevant to the core of the patch.

Reply:
oh sorry, i don't know how that got in there!